test_launch_ros/test/test_launch_ros/actions/test_node.py

```diff
@@ -160,10 +211,12 @@ def test_launch_node_with_parameter_dict(self):
         self._assert_launch_no_errors([node_action])
 
         # Check the expanded parameters (will be written to a file).
-        expanded_parameter_files = node_action._Node__expanded_parameter_files
-        assert len(expanded_parameter_files) == 1
-        with open(expanded_parameter_files[0], 'r') as h:
-            expanded_parameters_dict = yaml.load(h, Loader=yaml.FullLoader)
+        expanded_parameter_arguments = node_action._Node__expanded_parameter_arguments
+        assert len(expanded_parameter_arguments) == 1
+        file_path, is_file = expanded_parameter_arguments[0]
+        assert is_file
+        with open(file_path, 'r') as h:
+            expanded_parameters_dict = yaml.load(h)
```

Comment:
@ivanpauno why remove the `Loader=yaml.FullLoader` kwarg?

Reply:
1. I have deleted that by mistake.
2. We should not been using it ...
  The rcl yaml parameter parser cannot handle `!!python/tuple` AFAIR.
  We were using this loader because we're dumping parameters arrays as tuples and not as lists. We should be doing the later.

I will fix `1.`. We should fix `2.` in a follow up (I have to confirm this is actually a problem).

Reply:
Done in https://github.com/ros2/launch_ros/pull/137/commits/ef5209c49295b281b3039af9639734d1a1023d64.